packages/dds/tree/src/transaction/transaction.ts

```diff
@@ -3,10 +3,8 @@
  * Licensed under the MIT License.
  */
 
```

Comment:
The transaction module doesn't seem to make sense anymore. Can we move the Checkout interface somewhere else and delete this folder?

Reply:
I brought this up with Craig and we agreed this was the lesser evil because we expect more things to be added to this module in the future, and because a lot of documentation refers to this module and we'd have to change them then change them back when more things are added back here.